src/app/shared/formly/types/date-picker-field/date-picker-field.component.scss

```diff
@@ -0,0 +1,6 @@
+.input-group {
+  // hack to change the days of week color
+  ::ng-deep .ngb-dp-weekday {
+    color: #6c757d !important;
```

Comment:
I would not call it a hack to use `ng-deep`. We explicitly allowed it in the the Stylelint configuration (https://github.com/intershop/intershop-pwa/blob/develop/.stylelintrc.json#L25). This is fine this way.
What is actually a hack is the hard coded color.
You can use `@import 'variables';` in the beginning of the `.scss` file and with that you have access to all our variables (but not bootstrap variables directly). `$text-muted` maps to `$gray-600` so you can write `color: $gray-600 !important;`.
For such a gray value this might no be as important but if you want to work with the primary color for example you cannot hard code it here since it will be used in different themes that might have different primary colors.